docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md

```diff
@@ -0,0 +1,84 @@
+# ScrollableBase
+
+ScrollableBase is a common component that supports scrolling. You can add several views to the ScrollableBase component.
+You can use a mouse or finger on the touch screen to drag a ScrollableBase component.
+
+![ScrollableBase](./media/ScrollableBase.png)
+
+## Create with property
+
+To create a ScrollableBase using various properties, follow these steps:
+
+1. Create ScrollableBase using the default constructor:
+
+    ```cs
```

Comment:
As per our style guide, for language identification : 

Global: Please replace cs with csharp
Reference:  https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#supported-languages

Reply:
Done